Configurations/descrip.mms.tmpl

```diff
@@ -781,13 +781,10 @@ reconfigure reconf :
 
       if ($args{src} =~ /\.html$/) {
 	  my $title = basename($args{src}, ".html");
+	  my $pod = $args{generator}->[0];
 	  return <<"EOF";
-$args{src}: $args{generator}->[0]
-	pipe pod2html "--podroot=\$(SRCDIR)/doc" --htmldir=.. -
-		       --podpath=man1:man3:man5:man7 "--infile=\$<" -
-		      "--title=$title" -
-	| \$(PERL) -pe "s|href=""http://man\\.he\\.net/(man\d/[^""]+)(?:\\.html)?""|href=""../\$1.html|g;" -
-	> \$\@
+$args{src}: $pod
+	\$(PERL) \$(SRCDIR)/util/mkpod2html.pl -i $pod -o \$\@ -t "$title" -r "\$(SRCDIR)/doc"
```

Comment:
That doesn't work on VMS because of command line syntax differences.  It's fixed by insert this just below the definition of `$pod`:

``` perl
	my $mkpod2html = sourcefile('util', 'mkpod2html.pl');
```

... and then modifying this command like this:

``` make
	\$(PERL) $mkpod2html -i $pod -o \$\@ -t "$title" -r "\$(SRCDIR)/doc"
```

Reply:
thanks, fixup commit pushed.

Reply:
Any second now? :wink: